### DISCUSSION TITLE
failed to register layer: ApplyLayer exit status 1 stdout:  stderr: unlinkat /usr/share/nginx/html: invalid argument

### DISCUSSION THREAD

**Discussion by mcncyo** [shlinkio/discussions#29] [Help wanted]
everytime I try to pull docker image I am getting this error message

failed to register layer: ApplyLayer exit status 1 stdout:  stderr: unlinkat /usr/share/nginx/html: invalid argument

`docker pull shlinkio/shlink-web-client
Using default tag: latest
latest: Pulling from shlinkio/shlink-web-client
8921db27df28: Pull complete
83e90619bc2e: Pull complete
d52adec6f48b: Pull complete
10eb2ce358fa: Pull complete
a1be370d6a52: Pull complete
689b9959905b: Pull complete
c7a81ce22aac: Pull complete
862c0366d3dc: Extracting [==================================================>]     256B/256B
9465d09027b4: Download complete
2ca6d047741c: Download complete
af47852c3a2b: Download complete
failed to register layer: ApplyLayer exit status 1 stdout:  stderr: unlinkat /usr/share/nginx/html: invalid argument
`

**Comment by acelaya** [comment]
Can you try `docker pull shlinkio/shlink-web-client:stable`?

**Comment by acelaya** [comment]
Might be related https://github.com/moby/moby/issues/43576
